docs/rollup_bsn_contract_guide.md

```diff
@@ -0,0 +1,33 @@
+# Rollup BSN Contract Guide
+
+## Introduction
+
+The Rollup BSN contract runs on Babylon Genesis and tracks finality signatures for rollup blocks. It verifies who signed, detects double-signing, and reports misbehavior for slashing. This is the core contract rollups use to connect to Babylon’s Bitcoin staking protocol.
+
+## Why Deploy on Babylon
```

Comment:
We don't have to sell it, we are writing technical documentation to describe the procedure of deploying the contract to Babylon Genesis.